tap_exacttarget/client.py

```diff
@@ -73,7 +84,15 @@ def get_auth_stub(config):
                                        .format(config['tenant_subdomain']))
         LOGGER.info("Authentication URL is: %s", params['authenticationurl'])
         auth_stub = FuelSDK.ET_Client(params=params)
-        transport = HttpAuthenticated(timeout=int(config.get('request_timeout', 900)))
+
+        # Set request timeout with config param `request_timeout`.
+        config_request_timeout = config.get('request_timeout')
+        if config_request_timeout and float(config_request_timeout):
+            request_timeout = float(config_request_timeout)
+        else:
+            request_timeout = REQUEST_TIMEOUT # If value is 0,"0","" or not passed then it set default to 300 seconds.
```

Comment:
@harshpatel4crest Can we remove this as request_timeout is already calculated above in this function itself?

Reply:
Done, added single code change to set timeout value